modules/axhal/src/arch/aarch64/trap.rs

```diff
@@ -1,12 +1,11 @@
 use core::arch::global_asm;
 
+use super::TrapFrame;
```

Comment:
move `super::TrapFrame` to line 7.

Typically, references from this crate should be placed at the end.

Reply:
Thanks.I learn a lot.